crates/solver/src/arguments.rs

```diff
@@ -415,20 +415,16 @@ impl std::fmt::Display for Arguments {
             "max_additional_eden_tip: {}",
             self.max_additional_eden_tip
         )?;
-        writeln!(
-            f,
-            "max_submission_seconds: {:?}",
-            self.max_submission_seconds
-        )?;
+        writeln!(f, "max_submission: {:?}", self.max_submission)?;
```

Comment:
nit: Now it's no longer obvious that this is a time period. Maybe `max_submission_time`?